From bcb567214e8d5a6a7a25b8c65dbf0492eac68bd7 Mon Sep 17 00:00:00 2001
From: Semen Fomchenkov <armatik@altlinux.org>
Date: Thu, 5 Jun 2025 16:50:39 +0300
Subject: [PATCH] metainfo: fixed errors that cause appstream validation to
 fail

Due to the absence of the <p> tag in some places before the <ul> list,
appstream validation ended with an error.
---
 ...moussaoui.Authenticator.metainfo.xml.in.in | 40 +++++++++++++++++--
 1 file changed, 37 insertions(+), 3 deletions(-)

diff --git a/data/com.belmoussaoui.Authenticator.metainfo.xml.in.in b/data/com.belmoussaoui.Authenticator.metainfo.xml.in.in
index 87a22e82..8dfb3e6f 100644
--- a/data/com.belmoussaoui.Authenticator.metainfo.xml.in.in
+++ b/data/com.belmoussaoui.Authenticator.metainfo.xml.in.in
@@ -122,13 +130,28 @@
       </description>
     </release>
     <release version="4.1.5" date="2022-06-06">
-      <description>Fix a crash when loading an account token</description>
+      <description>
+        <p>Bugfixes release</p>
+        <ul>
+          <li>Fix a crash when loading an account token</li>
+        </ul>
+      </description>
     </release>
     <release version="4.1.4" date="2022-06-04">
-      <description>Fix fallback to host keyring and shell search provider results</description>
+      <description>
+        <p>Bugfixes release</p>
+        <ul>
+          <li>Fix fallback to host keyring and shell search provider results</li>
+        </ul>
+      </description>
     </release>
     <release version="4.1.3" date="2022-05-31">
-      <description>Fallback to host keyring in case the portal is not available</description>
+      <description>
+        <p>Bugfixes release</p>
+        <ul>
+          <li>Fallback to host keyring in case the portal is not available</li>
+        </ul>
+      </description>
     </release>
     <release version="4.1.2" date="2022-05-29">
       <description>
-- 
GitLab

